frontend/src/components/new-sightings-page/NewSightingForm.tsx

```diff
@@ -37,6 +37,10 @@ interface FormErrors {
   whaleCount: string;
   anyError: string;
 }
+const today = new Date();
+const future = new Date(today.setDate(today.getDate()))
+  .toISOString()
+  .slice(0, 16);
```

Comment:
This isn't the best way of doing this - I can't figure out by reading the code what you're trying to do (and a sliced string as you've done it here is very concerning!) Also `new Date(today.setDate(today.getDate()))` can't be right...

Install the [`date-fns`](https://date-fns.org/) package, and make use of the [`format`](https://date-fns.org/v2.29.3/docs/format) function which lets you easily format any date to the string of your wildest dreams:
```suggestion
const now: string = format(new Date(), "yyyy-MM-dd'T'HH:mm");
```
Then just use `now` below!